package.json

```diff
@@ -24,9 +24,11 @@
     "test:size": "bundlesize",
     "flow": "flow check",
     "flow:watch": "flow-watch",
+    "format": "eslint src --fix",
```

Comment:
Can you change `src` here to `./**/*.js` so that it targets the same files that lint-staged is targeting please?